### PR TITLE
[NeuralChat] Integrate photoai backend into restful API

### PR DESCRIPTION
**Pull request by letonghan**
## Type of Change

feature
API changed

## Description

Integrate the backend codes of photoai demo into neural chat restful API, including database utils and functional APIs.

## Expected Behavior & Potential Risk

Users are able to start photoai backend server through neural chat.

## How has this PR been tested?

Local test on server and remote test via Nginx.

## Dependency Change?

Add dependencies to requirements.txt

### CODE REVIEW

File: .github/workflows/script/unitTest/run_unit_test_neuralchat.sh [hunk 1]
```diff
@@ -73,6 +73,10 @@ function main() {
     apt-get update
     apt-get install ffmpeg -y
     apt-get install lsof
+    apt-get install libgl1
+    apt-get install -y libgl1-mesa-glx
+    apt-get install -y libgl1-mesa-dev
+    apt-get install libsm6 libxext6 -y
```
Comment (VincyZhang) [inline .github/workflows/script/unitTest/run_unit_test_neuralchat.sh:79]: should this installed in docker images? only for talking photo?

Comment (letonghan) [reply .github/workflows/script/unitTest/run_unit_test_neuralchat.sh:79]: The opencv package relies on libGL.so.1, so these packages need to be installed.

File: requirements.txt [hunk 2]
```diff
@@ -6,3 +6,6 @@ setuptools_scm[toml]>=6.2
 --extra-index-url https://download.pytorch.org/whl/cpu
 torch==2.1.0+cpu
 accelerate
+pymysql
```
Comment (VincyZhang) [inline requirements.txt:9]: why these are needed to add into requirements here?

**Comment by lvliang-intel** [comment]
Will add UT cases when we have resource.

File: intel_extension_for_transformers/neural_chat/examples/photo_ai/backend/photoai.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the parameter configuration file for NeuralChat Serving.
+
+#################################################################################
+#                             SERVER SETTING                                    #
+#################################################################################
+host: 127.0.0.1
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/examples/photo_ai/backend/photoai.yaml:23]: change to 0.0.0.0

File: intel_extension_for_transformers/neural_chat/pipeline/plugins/ner/ner.py [hunk 4]
```diff
@@ -41,11 +40,17 @@ class NamedEntityRecognition():
         Set bf16=True if you want to inference with bf16 model.
     """
 
-    def __init__(self, model_path="./Llama-2-7b-chat-hf/", spacy_model="en_core_web_lg", bf16: bool=False) -> None:
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/pipeline/plugins/ner/ner.py:44]: change to "meta-llama/Llama-2-7b-chat-hf"

File: intel_extension_for_transformers/neural_chat/examples/photo_ai/backend/photoai.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+port: 9000
+
+model_name_or_path: "./Llama-2-7b-chat-hf"
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/examples/photo_ai/backend/photoai.yaml:26]: change to "meta-llama/Llama-2-7b-chat-hf"

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**